### PR TITLE
gnuradio 3.8.1.0, gr-osmosdr 0.2.0

### PR DESCRIPTION
**Pull request by SeekingMeaning**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Previous attempts: #49733, #56628

Should we also add these lines to `def install` for `gr-osmosdr`?

```ruby
    site_packages = lib/"python#{xy}/site-packages"
    pth_contents = "import site; site.addsitedir('#{site_packages}')\n"
    (venv_root/"lib/python#{xy}/site-packages/homebrew-gr-osmosdr.pth").write pth_contents
```

### CODE REVIEW

File: Formula/gnuradio.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
 class Gnuradio < Formula
+  include Language::Python::Virtualenv
+
   desc "SDK providing the signal processing runtime and processing blocks"
   homepage "https://gnuradio.org/"
-  url "https://gnuradio.org/releases/gnuradio/gnuradio-3.7.13.4.tar.gz"
-  sha256 "c536c268b1e9c24f1206bbc881a5819ac46e662f4e8beaded6f3f441d3502f0d"
+  url "https://github.com/gnuradio/gnuradio/releases/download/v3.8.1.0/gnuradio-3.8.1.0.tar.gz"
```
Comment (SMillerDev) [inline Formula/gnuradio.rb:6]: Why are we not keeping their website?

**Comment by iMichka** [comment]
Cool. We will also be able to drop numpy@1.16 which was only used here if I remember well. Thanks for opening a PR with the undefined lookup patch/

File: Formula/gnuradio.rb (hunk 1 above)
Comment (Bo98) [reply Formula/gnuradio.rb:6]: Looks like they've stopped uploading new releases to their website.

Comment (SeekingMeaning) [reply Formula/gnuradio.rb:6]: Related: gnuradio/gnuradio#3672

**Comment by SeekingMeaning** [comment]
> We will also be able to drop numpy@1.16 which was only used here if I remember well.

`opencv@2` is the last dependent of `numpy@1.16`

**Comment by SMillerDev** [comment]
> `opencv@2` is the last dependent of `numpy@1.16`

Isn't that deprecated yet?

**Comment by SeekingMeaning** [comment]
Yes, although it still gets installed quite a bit https://formulae.brew.sh/formula/opencv@2

File: Formula/gnuradio.rb (hunk 1 above)
Comment (SeekingMeaning) [reply Formula/gnuradio.rb:6]: > @SeekingMeaning Thanks for pointing that out. We'll get the release artifacts copied over to there.

Will re-resolve this once updated

**Comment by jonchang** [comment]
@SeekingMeaning is this ready to merge?

**Review by fxcoudert — APPROVED** [review]
Looking good to me, we should merge because the current version is missing bottles and fails to build from source (https://github.com/Homebrew/homebrew-core/issues/58983)

And it's so great to finally move this one to Python 3 👍

**Comment by icpz** [comment]
Could you please enable grc?

**Comment by SMillerDev** [comment]
@icpz please make a pull request to enable it if you need it

**Pull request closed without merging**